src/main/java/com/cloudogu/smeagol/wiki/domain/Message.java

```diff
@@ -0,0 +1,57 @@
+package com.cloudogu.smeagol.wiki.domain;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+
+import java.util.Objects;
+
+/**
+ * Message is a simple non empty string value object.
+ */
+public final class Message {
+
+    private final String value;
+
+    private Message(String value) {
+        this.value = value;
+    }
+
+    /**
+     * Returns the string representation of the message.
+     *
+     * @return string representation
+     */
+    public String getValue() {
+        return value;
+    }
+
+    /**
+     * Creates a message from its string representation.
+     *
+     * @param value string representation of message
+     *
+     * @return message value object
+     */
+    public static Message valueOf(String value) {
+        Preconditions.checkArgument(!Strings.isNullOrEmpty(value), "message is null or empty");
+        return new Message(value);
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS00122)